src/pages/css/App.css

```diff
@@ -1,3 +1,14 @@
+*,
```

Comment:
Bu joyga nega tegdin, orqaga qaytar

Reply:
Bu narsa kere
For more info: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing)

Reply:
Bu bugda yogidiyu. Bug bomagani nega tiyvosan

Reply:
Bu yaxshilash uchun